mm2src/coins/lp_coins.rs

```diff
@@ -1127,7 +1128,7 @@ impl TradePreimageError {
                 let available = big_decimal_from_sat_unsigned(sum_utxos, decimals);
                 let required = big_decimal_from_sat_unsigned(required, decimals);
                 TradePreimageError::NotSufficientBalance {
-                    coin,
+                    coin: platform_ticker,
```

Comment:
First of all, `platform_ticker` needs to be used instead of `coin` in all cases, not in `GenerateTxError::NotEnoughUtxos` only.
Also I think it's not really common case when a UTXO coin is a platform coin. Currently it's true for `QRC20` and `SLP` tokens.
Summing up, I think we don't need to take `platform_ticker` **and** `coin` both. But we need to pass `coin.platform_ticker()` to the `TradePreimageError:: from_generate_tx_error` function instead of `coin.ticker()` **when it's required**, i.e. for `QRC20` and `SLP` tokens.